tests/moduletool/test_build.py

```diff
@@ -257,3 +257,64 @@ def test_create_dev_build_of_pre_tagged_module(tmpdir, modules_dir: str, explici
         assert re.search(r"1.2.3\.dev[0-9]{14}", path_to_wheel)
     else:
         assert "1.2.3.dev0" in path_to_wheel
+
+
+@pytest.mark.parametrize_any(
+    "module_name, is_empty",
+    [
+        ("minimalv1module", True),
+        ("minimalv1module", False),
+        ("elaboratev1module", True),
+        ("elaboratev1module", False),
+    ],
+)
+def test_build_module_existing_plugin_dir(tmpdir, modules_dir: str, module_name, is_empty) -> None:
```

Comment:
I would think this is a `slowtest`, not sure.

Reply:
I takes around 4 seconds to run the 4 tests on my computer so I added the annotation